cmake/modules/hunter_install_libsodium_config.cmake

```diff
@@ -0,0 +1,25 @@
+include(CMakePackageConfigHelpers)
+include(hunter_test_string_not_empty)
+
+function(hunter_install_libsodium_config)
+
+    hunter_test_string_not_empty("${HUNTER_libsodium_VERSION}")
+    hunter_test_string_not_empty("${HUNTER_PACKAGE_INSTALL_PREFIX}")
+    hunter_test_string_not_empty("${HUNTER_SELF}")
+
+    set(templates "${HUNTER_SELF}/cmake/templates")
+    set(configs "${HUNTER_PACKAGE_INSTALL_PREFIX}/lib/cmake/libsodium")
+
+    # Forward some variables
+    if(BUILD_SHARED_LIBS)
```

Comment:
Oh I see, not sure haha, was looking at what you did for boost to forward the variables